proto-lens-protoc/proto-lens-protoc.cabal

```diff
@@ -14,7 +17,14 @@ build-type:          Simple
 cabal-version:       >=1.21
 
 library
-  exposed-modules:   Data.ProtoLens.Setup
+  exposed-modules:
```

Comment:
It's unfortunate that the current Cabal integration approach will cause any library that uses a .proto file to link against all these modules as well as haskell-src-exts.  [For example.](https://github.com/google/proto-lens/blob/master/proto-lens-tests/proto-lens-tests.cabal#L73)

Not sure if there's a better organization of libraries/modules.  What you have is fine for now.   Maybe the right solution is `custom-setup` clauses, though that's only on newer Cabal and support is somewhat spotty (e.g., https://github.com/commercialhaskell/stack/issues/2094).